### PR TITLE
rm timeout override from getContainerEnv

### PR DESCRIPTION
**Pull request by dgarberj**
default is 30000ms as noted here: https://github.com/Azure/Bridge-To-Kubernetes/blob/main/src/common/Kubernetes/IKubectlImpl.cs#L26

temporarily to fix my blocker here:
https://github.com/Azure/Bridge-To-Kubernetes/issues/179

### CODE REVIEW

File: src/common/Kubernetes/KubernetesClient.cs [hunk 1]
```diff
@@ -1015,9 +1015,7 @@ public async Task<V1RoleBinding> CreateOrReplaceV1RoleBindingInNamespaceAsync(V1
                     $"exec {podName} -c {containerName} -n {namespaceName} -- env",
                     onStdOut: outputHandler,
                     onStdErr: (string error) => errorSb.Append(error),
-                    cancellationToken,
-                    timeoutMs: (int)TimeSpan.FromSeconds(10).TotalMilliseconds);
```
[the remark this answers is not in the document]
Comment (dgarberj) [reply src/common/Kubernetes/KubernetesClient.cs:1019]: I apologize, I am not familiar with this environment. I don't think I'll be able to add the UTs

[the remark this answers is not in the document]
Comment (hsubramanianaks) [reply src/common/Kubernetes/KubernetesClient.cs:1019]: @dgarberj That's okay, we will move this to testing and then I will do another PR for adding UTs and refactor this.

**Review by hsubramanianaks — DISMISSED** [review]
Looks good to me.

**Review by hsubramanianaks — DISMISSED** [review]
looks good.

**Review by hsubramanianaks — APPROVED** [review]
looks good.

**Comment by dgarberj** [comment]
Thank you for your help with getting this approved. Just curious, what does the release timeline look like? i.e. How can I use the latest build containing this fix?

**Comment by dgarberj** [comment]
Yeah that would be great! I am unfamiliar with the vsix build process so that would be very helpful. Sent from my iPhoneOn Apr 14, 2023, at 4:04 PM, Hariharan Subramanian ***@***.***> wrote:﻿

Thank you for your help with getting this approved. Just curious, what does the release timeline look like? i.e. How can I use the latest build containing this fix?

@dgarberj we can plan for some testing along with other features, fixes and will be available to be released next week. we do release every other week or on need basis, no strict cadence followed.  I have other customers waiting for some other fixes so will plan a release next week. Thanks for the patience.
Meanwhile I can give you a VSIX with the fix in it which you can use until official release ? Please let me know.

**Comment by hsubramanianaks** [comment]
> Yeah that would be great! I am unfamiliar with the vsix build process so that would be very helpful. Sent from my iPhoneOn Apr 14, 2023, at 4:04 PM, Hariharan Subramanian ***@***.***> wrote:﻿ Thank you for your help with getting this approved. Just curious, what does the release timeline look like? i.e. How can I use the latest build containing this fix? @dgarberj we can plan for some testing along with other features, fixes and will be available to be released next week. we do release every other week or on need basis, no strict cadence followed. I have other customers waiting for some other fixes so will plan a release next week. Thanks for the patience. Meanwhile I can give you a VSIX with the fix in it which you can use until official release ? Please let me know. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

@dgarberj Hi here is the VSIX to test - https://github.com/Azure/Bridge-To-Kubernetes/releases/tag/2.0.120230418

i released it as beta release , please set BRIDGE_ENVIRONMENT='dev' before installing and running this VSIX, thank you.

**Pull request merged**